tools/ci/run_tc.py

```diff
@@ -161,9 +161,14 @@ def install_webkitgtk_from_apt_repository(channel):
 
 
 def install_webkitgtk_from_tarball_bundle(channel):
-    run(["wget", "https://webkitgtk.org/built-products/nightly/webkitgtk-nightly-build-last.tar.xz",
-        "-O", "/tmp/webkitgtk-nightly-build-last.tar.xz"])
-    run(["sudo", "tar", "xfav", "/tmp/webkitgtk-nightly-build-last.tar.xz", "-C", "/"])
+    resp = urlopen("https://webkitgtk.org/built-products/nightly/webkitgtk-nightly-build-last.tar.xz")
+    with open("/tmp/webkitgtk-nightly-build-last.tar.xz", "wb") as f:
```

Comment:
I'm not sure if /tmp/ is always writable, but would it make sense to use https://docs.python.org/2/library/tempfile.html#tempfile.NamedTemporaryFile to not have to worry about it?

Reply:
AFAIK `/tmp` is assumed to be always writable. Actually `tempfile.NamedTemporaryFile` returns a file on `/tmp`. But now that you mention it, I see that I'm not deleting the temporal file on `/tmp` after unpacking it. And I can have python do that automatically when the file is closed by just using `tempfile.NamedTemporaryFile`. Let's try it.

Reply:
Pushed d601996c19ab6afced2eb82179d97313fbd20a95 with the ```tempfile.NamedTemporaryFile``` approach